### PR TITLE
flag: Log all 2014 Listing Edit activity as questionable

### PR DESCRIPTION
**Pull request by kekey1**
This depends on Andrew's 2176 branch because that is where the feature flag is first created in our code. Should not be merged until after his PR is merged.

[OCD-2178]

### CODE REVIEW

File: chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertificationIdController.java [hunk 1]
```diff
@@ -49,7 +51,10 @@
 
     @Autowired
     private CertificationIdManager certificationIdManager;
-    
+
+    @Autowired
+    private FF4j ff4j;
```
Comment (tmy1313) [inline chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertificationIdController.java]: Can you update this class to use constructor injection?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/questionableactivity/ListingQuestionableActivityProvider.java [hunk 2]
```diff
@@ -26,6 +29,9 @@
 @Component
 public class ListingQuestionableActivityProvider {
 
+    @Autowired
+    private FF4j ff4j;
```
Comment (tmy1313) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/questionableactivity/ListingQuestionableActivityProvider.java]: Constructor injection?

File: chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertificationIdController.java (hunk 1 above)
Comment (kekey1) [reply chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertificationIdController.java]: Addressed.

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/questionableactivity/ListingQuestionableActivityProvider.java (hunk 2 above)
Comment (kekey1) [reply chpl/chpl-service/src/main/java/gov/healthit/chpl/questionableactivity/ListingQuestionableActivityProvider.java]: Addressed.

**Pull request merged**